unstructured/utils.py

```diff
@@ -21,13 +22,33 @@
     cast,
 )
 
+import magic
+import matplotlib.pyplot as plt
 import requests
+from matplotlib import colors, patches
+from pdf2image import convert_from_path
+from PIL import Image
 from typing_extensions import ParamSpec
 
 from unstructured.__version__ import __version__
 
 DATE_FORMATS = ("%Y-%m-%d", "%Y-%m-%dT%H:%M:%S", "%Y-%m-%d+%H:%M:%S", "%Y-%m-%dT%H:%M:%S%z")
-
+TYPE_TO_COLOUR_MAP = {
```

Comment:
I remember some of these colors are difficult to see/read (like cyan), would like to collapse all these colors into a list of "eye-popping" colors. In the end this is for understanding an output, not exactly to identify classes. For that, I would add a string at the start of the text inside the bbox.

Reply:
Oh I tried to pick the most eye-popping colours without repetition. I changed `cyan` for `indigo`. You can always set `TYPE_TO_COLOUR_MAP` to diff values and see how it gets. There is a string with the label and element-index also being drawn per element. Is this what you mean? let me know if you consider there are more necessary changes to finally merge this PR.